lib/node/node.ml

```diff
@@ -6,9 +6,7 @@ module Client = Client
 module Failure_detector = Failure_detector
 module Inbox = Inbox
 
-type 'a t = 'a Types.node
-
-let init ~state ?(router = fun m -> m) ~msg_handler ?(init_peers = [])
+let init ?(router = fun m -> m) ~msg_handler ?(init_peers = [])
```

Comment:
@rosalogia I first tried to switch to `?state` but can't figure out how to make it works.
Any help will be welcomed 🙏